src/example.pl

```diff
@@ -6,3 +6,9 @@
 foreach my $ele (@array) {
     print "$ele\n";
 }
+
+my @array2 = ("Nyan", "Cat");
```

Comment:
**[perlcritic]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
**Useless interpolation of literal string**, near <code>"Nyan", "Cat"</code>.<br>(Ref: [ValuesAndExpressions::ProhibitInterpolationOfLiterals](https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitInterpolationOfLiterals))